pkg/services/contexthandler/contexthandler.go

```diff
@@ -112,16 +112,16 @@ func (h *ContextHandler) Middleware(next http.Handler) http.Handler {
 			reqContext.Logger = reqContext.Logger.New("traceID", traceID)
 		}
 
-		identity, err := h.authnService.Authenticate(ctx, &authn.Request{HTTPRequest: reqContext.Req, Resp: reqContext.Resp})
+		id, err := h.authnService.Authenticate(ctx, &authn.Request{HTTPRequest: reqContext.Req, Resp: reqContext.Resp})
```

Comment:
we can't use the variable "identity" and the package import (without adding an alias) -- id seems reasonable, but there may be some better alternative

Reply:
Yeah we usually have the same struggle in other places. I think we have used id there as well :+1: